File: test_data/code/python/1880-check-if-word-equals-summation-of-two-words.py
class Solution:
    def isSumEqual(self, firstWord: str, secondWord: str, targetWord: str) -> bool:
        def get_numerical_value(word: str) -> int:
            return int(''.join(str(ord(c) - ord('a')) for c in word))

        return get_numerical_value(firstWord) + get_numerical_value(secondWord) == get_numerical_value(targetWord)

File: test_data/code/java/1880-check-if-word-equals-summation-of-two-words.java
class Solution {
    public boolean isSumEqual(String firstWord, String secondWord, String targetWord) {
        int sum = 0;
        int target = 0;
        
        for(int i = 0; i < firstWord.length(); i++){
            sum = sum * 10 + (firstWord.charAt(i) - 'a');
        }
        for(int i = 0; i < secondWord.length(); i++){
            sum = sum + (secondWord.charAt(i) - 'a');
        }
        
        for(int i = 0; i < targetWord.length(); i++){
            target = target * 10 + (targetWord.charAt(i) - 'a');
        }
        
        return sum == target;
    }
}